### ISSUE TITLE
Vulnerabilities in flyway command line 9.40

### ISSUE THREAD

**Issue by jmcruz1983** [flyway/flyway#3538]
Following vulnerabilities are being reported by docker image scans:

https://github.com/flyway/flyway-docker/issues/110

```
Java (jar)
==========
Total: 4 (UNKNOWN: 0, LOW: 0, MEDIUM: 1, HIGH: 2, CRITICAL: 1)

+---------------------------------------------+------------------+----------+-------------------+--------------------------------+---------------------------------------+
|                   LIBRARY                   | VULNERABILITY ID | SEVERITY | INSTALLED VERSION |         FIXED VERSION          |                 TITLE                 |
+---------------------------------------------+------------------+----------+-------------------+--------------------------------+---------------------------------------+
| com.fasterxml.jackson.core:jackson-databind | CVE-2022-42003   | HIGH     | 2.13.2.1          | 2.14.0-rc1                     | In FasterXML jackson-databind before  |
|                                             |                  |          |                   |                                | 2.14.0-rc1, resource exhaustion c ... |
|                                             |                  |          |                   |                                | -->avd.aquasec.com/nvd/cve-2022-42003 |
+                                             +------------------+          +                   +--------------------------------+---------------------------------------+
|                                             | CVE-2022-42004   |          |                   | 2.13.4                         | In FasterXML jackson-databind before  |
|                                             |                  |          |                   |                                | 2.13.4, resource exhaustion can o ... |
|                                             |                  |          |                   |                                | -->avd.aquasec.com/nvd/cve-2022-42004 |
+---------------------------------------------+------------------+----------+-------------------+--------------------------------+---------------------------------------+
| com.google.protobuf:protobuf-java           | CVE-2022-3171    | MEDIUM   | 3.21.4            | 3.16.3, 3.19.6, 3.20.3, 3.21.7 | [potential denial of service issue    |
|                                             |                  |          |                   |                                | in the Java Protobuf runtime]         |
|                                             |                  |          |                   |                                | -->avd.aquasec.com/nvd/cve-2022-3171  |
+---------------------------------------------+------------------+----------+-------------------+--------------------------------+---------------------------------------+
| org.hsqldb:hsqldb                           | CVE-2022-41853   | CRITICAL | 2.6.1             | 2.7.1                          | Those using java.sql.Statement        |
|                                             |                  |          |                   |                                | or java.sql.PreparedStatement         |
|                                             |                  |          |                   |                                | in hsqldb ...                         |
|                                             |                  |          |                   |                                | -->avd.aquasec.com/nvd/cve-2022-41853 |
+---------------------------------------------+------------------+----------+-------------------+--------------------------------+---------------------------------------+
```

**Comment by jakmal521** [comment]
I would add upgrading org.apache.commons:commons-text to version 1.10.0  due to https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2022-42889 as well

**Comment by DoodleBobBuffPants** [comment]
Thanks for raising this issue

We use a vulnerability scanner internally to identify dependencies we can upgrade, and some of these will be fixed in the next release

Not every vulnerability however has a fix. For example, the fixed version of hsqldb [isn't released yet](https://mvnrepository.com/artifact/org.hsqldb/hsqldb) but we will update it when it becomes available

**Comment by jmcruz1983** [comment]
Thanks for reply @DoodleBobBuffPants 
btw, seems that [hsqldb](https://mvnrepository.com/artifact/org.hsqldb/hsqldb) released 2.7.1 that is vulnerability free
